### PR TITLE
Add spectral networking and graphml export to wizard

### PR DESCRIPTION
**Pull request by robinschmid**
Need to merge #1451 first.

### CODE REVIEW

**Comment by robinschmid** [comment]
All ready from my side

**Comment by robinschmid** [comment]
Done @SteffenHeu

File: src/main/java/io/github/mzmine/modules/io/export_network_graphml/NetworkGraphMlExportTask.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  }
+
+  @Override
+  public double getFinishedPercentage() {
+    return progress;
+  }
+
+  @Override
+  public void run() {
+    setStatus(TaskStatus.PROCESSING);
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/io/export_network_graphml/NetworkGraphMlExportTask.java:78]: No applied method

File: src/main/java/io/github/mzmine/modules/visualization/networking/visual/NetworkLayoutComputeTask.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    Layout layout = new SpringBox(false, new Random(42));
+    layout.setForce(1);
+    // less precision for large networks
+    double stab = size < 200 ? 0.9 : 0.75;
+
+    logger.fine("Layout of %d nodes, stabilization at %.2f".formatted(size, stab));
+
+    layout.setQuality(stab);
+    // below taken from
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/visualization/networking/visual/NetworkLayoutComputeTask.java:79]: taken from what?

**Review by SteffenHeu — COMMENTED** [review]
only a small comment

File: src/main/java/io/github/mzmine/modules/visualization/networking/visual/NetworkLayoutComputeTask.java (hunk 2 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/visualization/networking/visual/NetworkLayoutComputeTask.java]: added javadoc

**Pull request merged**